### DISCUSSION TITLE
Build a chatbot with huggingFace

### DISCUSSION THREAD

**Discussion by Biigode** [huggingface/huggingface.js#681] [Q&A] [answered]
Hi,

I want to build a chatbot using hf.conversation. There is a reference to it in the documentation, but I can't find it in the library. Can you help me?

![hf chatCompletion](https://github.com/huggingface/huggingface.js/assets/36681261/fcc1cc5f-a6ba-4c15-b21e-071764f35d35)

**Comment by coyotte508** [comment, marked as answer]
It's `hf.chatCompletion`.

You can also do directly `import { chatCompletion } from "@huggingface/inference"`

Make sure you have a recent version of `@huggingface/inference` installed, it's been added in the last few weeks. 

Can you share more details around your error?

**Reply by Biigode** [reply]
Thank you for your assistance.
I was using version 2.6.7, and after updating to version 2.7.0, everything works perfectly.

![chatCompletion](https://github.com/huggingface/huggingface.js/assets/36681261/9bfba05c-ffa4-4df7-a403-d6f0a630f0f7)
